documentation/modules/proc-accessing-kafka-using-routes.adoc

```diff
@@ -40,6 +45,9 @@ spec:
   zookeeper:
     # ...
 ----
++
+WARNING: An OpenShift Route address comprises the name of the Kafka cluster, the name of the listener, and the name of the namespace it is created in.
+For example, `my-cluster-external-myproject`. Be careful with naming as the whole length of the address must not exceed 63 characters, with an 11-character limit for the listener name.
```

Comment:
I think if we want the example, we should use `my-cluster-kafka-listener1-bootstrap-myproject` to make it more corresponding ... this would be the longest route name ... `<cluster-name>-kafka-<listener-name>-bootstrap-<namespace>`.

Reply:
Okay. I've changed the examples in the metadata and updated to:

WARNING: An OpenShift Route address comprises the name of the Kafka cluster, the name of the listener, and the name of the namespace it is created in. For example, `my-cluster-kafka-listener1-bootstrap-myproject` (_CLUSTER-NAME_-kafka-_LISTENER-NAME_-bootstrap-_NAMESPACE_). Be careful that the whole length of the address does not exceed a maximum limit of 63 characters.